cli/cmd/install.go

```diff
@@ -259,3 +273,43 @@ func checkIstioInstallation() error {
 
 	return nil
 }
+
+func getAPIEndpoint(keptnNamespace string, serviceType string) (string, error) {
+	var endpoint, port string
+	if serviceType == "NodePort" {
+		external, err := keptnutils.ExecuteCommand("kubectl", []string{"get", "nodes", "-o", "jsonpath='{ $.items[0].status.addresses[?(@.type==\"ExternalIP\")].address }'"})
+		internal, err := keptnutils.ExecuteCommand("kubectl", []string{"get", "nodes", "-o", "jsonpath='{ $.items[0].status.addresses[?(@.type==\"InternalIP\")].address }'"})
+		if err != nil {
+			return "", err
+		}
+		endpoint = strings.Trim(external, "'")
+		internal = strings.Trim(internal, "'")
+		port, _ = keptnutils.ExecuteCommand("kubectl", []string{"get", "svc", "api-gateway-nginx", "-n", keptnNamespace, "-o", "jsonpath='{.spec.ports[?(@.name==\"http\")].nodePort}'"})
+		port = strings.Trim(port, "'")
+		if endpoint == "" {
+			endpoint = internal
+		}
+	} else if serviceType == "LoadBalancer" {
+		external, err := keptnutils.ExecuteCommand("kubectl", []string{"get", "svc", "api-gateway-nginx", "-n", keptnNamespace, "-o", "jsonpath='{.status.loadBalancer.ingress[0].ip}'"})
+		if err != nil {
+			return "", err
+		}
+		endpoint = strings.Trim(external, "'")
+		port = "80"
+	}
+	return "http://" + endpoint + ":" + port + "/api", nil
```

Comment:
Please add some docs to this code (e.g., fetching external and internal node IP, getting port of api-gateway-nginx service)